mathesar/tests/conftest.py

```diff
@@ -69,3 +69,37 @@ def paste_filename():
 @pytest.fixture(scope='session')
 def headerless_csv_filename():
     return 'mathesar/tests/data/headerless_patents.csv'
+
+
+@pytest.fixture(scope='session')
+def patents_url():
```

Comment:
Wouldn't these fixtures make more sense in the `test_data_file_api.py` file? I'm not sure that they will be reused elsewhere.

Reply:
The URL and filename is needed for table tests, but I moved the specific data and mock fixtures to `test_data_file_api` :+1: